cockatrice/src/dlg_edit_password.cpp

```diff
@@ -0,0 +1,60 @@
+#include <QSettings>
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QDebug>
+
+#include "dlg_edit_password.h"
+
+DlgEditPassword::DlgEditPassword(QWidget *parent)
+    : QDialog(parent)
+{
+    QSettings settings;
+    settings.beginGroup("server");
+
+    oldPasswordLabel = new QLabel(tr("Old password:"));
+    oldPasswordEdit = new QLineEdit(settings.value("password").toString());
+    oldPasswordLabel->setBuddy(oldPasswordEdit);
+    oldPasswordEdit->setEchoMode(QLineEdit::Password);
+
+    newPasswordLabel = new QLabel(tr("New password:"));
```

Comment:
Can you make a second field so that the user is forced to type twice and confirm that the password is correct?